openjdk/playlist.xml

```diff
@@ -115,9 +115,16 @@
 		<groups>
 			<group>openjdk</group>
 		</groups>
-		<impls>
-			<impl>hotspot</impl>
-		</impls>
+		<disables>
+			<disable>
+				<comment>https://github.com/adoptium/aqa-tests/issues/5357</comment>
+				<impl>openj9</impl>
+			</disable>
+			<disable>
+				<comment>https://github.com/adoptium/aqa-tests/issues/5357</comment>
+				<impl>ibm</impl>
+			</disable>
+		</disables>
```

Comment:
Can we move `<disables>` right after `<testCaseName>`? It will be aligned with the others and easy to read. Thank you.

Reply:
Updated. Thanks.